README.md

```diff
@@ -153,76 +166,82 @@ local params = {
     "TopicArn": "YOUR_TOPIC_ARN_HERE"
```

Comment:
See comment re: updating constants in the constructor. This example should either use the constant defined in the constructor or should format the value in a similar way to the value of the constant's. 
```
const AWS_SNS_TOPIC_ARN = "< YOUR_TOPIC_ARN_HERE>";
local params = {
    "TopicArn": AWS_SNS_TOPIC_ARN
}
```
or 
```
local params = {
    "TopicArn": "< YOUR_TOPIC_ARN_HERE>"
}
```